rebar.config

```diff
@@ -22,8 +22,9 @@
 
 {deps, [
         {ranch, {git, "https://github.com/ninenines/ranch.git", {tag, "1.8.0"}}},
-        {ebloom,{git, "https://github.com/nhs-riak/ebloom.git", {branch, "nhse-develop"}}},
-        {riak_kv, {git, "https://github.com/nhs-riak/riak_kv.git", {branch, "nhse-develop"}}}
+        {ebloom, {git, "https://github.com/nhs-riak/ebloom.git", {branch, "nhse-develop"}}},
+        {folsom, {git, "https://github.com/nhs-riak/folsom.git", {branch, "nhse-develop-3.4"}}},
+        {riak_kv, {git, "https://github.com/nhs-riak/riak_kv.git", {branch, "nhse-d34-otp26"}}}
```

Comment:
possibly a temporary branch to point to?

Reply:
Now references updated permanent branch